### PR TITLE
Admissions Trial Project

### PR DESCRIPTION
**Pull request by batoorsohail**
# Admissions Trial Project:
We added:
- HTML file
- CSS file

### CODE REVIEW

**Review by batoorsohail — COMMENTED** [review]
# Changes required ♻️ 

### suggestions 
- It is better to add a remdme file that explains your project
- Add Linters to your project

Please check my comments and make changes to them.

Submit for re-review when you're done.

Thanks 😄

File: index.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+          
+        </div>
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#features">Features</a></li>
+            <li><a class="nav-link" href="#how-it-works">How it works</a></li>
+            <li><a class="nav-link" href="#pricing">Pricing</a></li>
+          </ul>
+          
+        </nav>
```
Comment (OyePriscilla) [inline index.html:29]: - [ ] Actually, I think it is still ok if we have `nav` tag wrapped with the `header` tag. The <nav> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a section of a page whose purpose is to provide navigation links. 👍

**Review by OyePriscilla — COMMENTED** [review]
Hi @batoorsohail   😃 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Good use of inline comments :heavy_check_mark: 
Clear and polite suggestion ✔️ 

## Required Changes ♻️

- [ ] Kindly please ensure that your comments come under the summary template; after you've done reviewing just click on "Finish your review".

205592593-523eae78-6d43-4103-b886-d38f7fcdec71

- [ ] [optional] It would be great to use a couple of emojis and/or gifs in the review. They make the code review more human-friendly. 🧑‍🤝‍🧑 

Kindly check the inline comments below as well.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by batoorsohail — COMMENTED** [review]
Hi, dear 👋,
Good job so far 👍 
There are some issues that you still need to work on.

so-close


## Required changes ♻️

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+@import 'https://fonts.googleapis.com/css?family=Lato:400,700';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {}
```
Comment (batoorsohail) [inline css/style.css:11]: I think it is better to delete .container because it doesn't contain any style.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+
+.container {
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.container {}
+
+.container {}
```
Comment (batoorsohail) [inline css/style.css:41]: I think it is better to delete .container because you have used it once at the top. 😉

File: index.html [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+          <br>
+          <input id="submit" type="submit" value="Get Started" class="btn">
+        </form>
+      </section>
+      <div class="container">
+        <section id="features">
+          <div class="grid">
+            <div class="icon">
+              <i class="fa fa-3x fa-fire">
+                ::before
```
Comment (batoorsohail) [inline index.html:45]: I think the ": : before" is not necessary, and you can delete it to make your code clean.

**Review by batoorsohail — COMMENTED** [review]
Hi, dear 👋,
Good job so far 👍 
There are some issues that you still need to work on.

so-close


## Required changes ♻️

**Review by Owusu-Desmond — APPROVED** [review]
Hi @batoorsohail 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Good to use inline comments ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Good to add a summary to your review
📌 Good Communication ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**